Client/Frontend/Toolbar+URLBar/TabToolbar.swift

```diff
@@ -131,15 +135,17 @@ extension TabToolbar: NotificationThemeable, PrivateModeUI {
     func applyTheme() {
         backgroundColor = UIColor.theme.ecosia.barBackground
         helper?.setTheme(forButtons: actionButtons)
-
+        
+        /* Ecosia: Remove private mode badge
         privateModeBadge.badge.tintBackground(color: UIColor.theme.ecosia.barBackground)
         privateModeBadge.backdrop.backgroundColor = UIColor.theme.ecosia.personalCounterSelection
-        privateModeBadge.backdrop.alpha = 1
+        privateModeBadge.backdrop.alpha = 1*/
         appMenuBadge.badge.tintBackground(color: UIColor.theme.ecosia.barBackground)
         warningMenuBadge.badge.tintBackground(color: UIColor.theme.ecosia.barBackground)
     }
 
     func applyUIMode(isPrivate: Bool) {
```

Comment:
technically you could comment the entire method 🤔

Reply:
I thought so as well, but this would make it no longer comply to the `PrivateModeUI`, which would then require some additional removals on the `BrowserViewController` that I though might make things more complex (both for if we want to add additional private mode ui in the future and for conflicts with Firefox code). Would you still say it is better to remove as much as possible?

Reply:
ah ok - good to know. I agree to not overcomplicate the issue. Fine as it is 👍